dev/reference/tools/data-migration/usage-scenarios/best-practice-dm-shard.md

```diff
@@ -0,0 +1,128 @@
+---
+title: 分表合并数据迁移最佳实践
+summary: 使用 DM 对分库分表进行合并迁移时的最佳实践。
+category: reference
+---
+# 分表合并数据迁移最佳实践
+
+本文目的是阐述在使用 DM 对分库分表进行合并迁移的场景中，DM 相关功能的支持限制，旨在给出一个业务的最佳实践。
+
+## 数据迁移任务
+
+在[分库分表合并同步的实现原理部分](/dev/reference/tools/data-migration/features/shard-merge.md#实现原理)，我们介绍了 shard group 的概念，简单来说可以理解为所有需要合并到同一个下游表的所有上游表即组成一个 shard group。
+
+当前的 shard DDL 算法为了能协调在不同分表执行 DDL 对 schema 变更的影响，加入了一些[使用限制](/dev/reference/tools/data-migration/features/shard-merge.md#使用限制)。而当这些使用限制由于某些异常原因被打破时，我们需要[手动处理 Sharding DDL Lock](/dev/reference/tools/data-migration/features/manually-handling-sharding-ddl-locks.md) 甚至是完整重做整个数据迁移任务。
+
+因此，为了减小异常发生时对数据迁移的影响，我们推荐将每一个 shard group 拆分成一个独立的数据迁移任务。这样当异常发生时，可能只有少部分迁移任务需要进行手动处理，其他数据迁移任务可以不受影响。
+
+## 手动处理 shard DDL lock
+
+从[分库分表合并同步的实现原理部分](/dev/reference/tools/data-migration/features/shard-merge.md#实现原理)我们可以知道，DM 中的 shard DDL 是用于协调不同上游分表向下游执行 DDL 的一种机制，本身并不是异常。
```

Comment:
```DM 中的 shard DDL 是用于协调不同上游分表向下游执行 DDL 的一种机制，本身并不是异常。```
->
```DM 中的 shard DDL lock 是用于协调不同上游分表向下游执行 DDL 的一种机制，本身并不是异常。```

Reply:
done in https://github.com/pingcap/docs-cn/pull/1847/commits/b4b6f130d30c824286404fa31b493c176e7a6e12.